Gopkg.toml

```diff
@@ -97,10 +103,18 @@
   name = "gopkg.in/olivere/elastic.v5"
   version = "^5.0.53"
 
- [[constraint]]
+[[constraint]]
   name = "github.com/gocql/gocql"
   branch = "master"
 
+[[constraint]]
+  name = "github.com/gogo/googleapis"
+  revision = "b23578765ee54ff6bceff57f397d833bf4ca6869"
+
+[[constraint]]
+  name = "github.com/gogo/protobuf"
+  revision = "fd9a4790f3963525fb889cc00e0a8f828e0b3a29"
```

Comment:
I'll merge it on green build, but we should figure out how to pin to versions instead of raw commit hash. E.g. `gogo/googleapis` has a 1.1.0 release in August, which is later than the commit we have.  And `gogo/protobuf` has pretty frequent releases.

Reply:
https://github.com/jaegertracing/jaeger/issues/1262

Reply:
It should work with newer versions too. This is copied from the old glide.yaml.